meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md

```diff
@@ -0,0 +1,1250 @@
+---
+## Standup meeting - 2018-10-30
+
+### Updates since last time
+
+##### @hsanjuan
+
+Last week:
+
+* Docker compose
+* Finished Batching removal
+* Twitter pinbot upgrade
+* Fix metrics PR
+* Chagngelog and doc upgrades for 0.7.0
+* libp2p bubbbling and others
+* Race conditions debugging
+* 0.7.0-rc1
+
+This week:
+
+
+* Test 0.7.0
+* Update pinbots
+* Release stable
+* CRDTs
+
+
+##### @lanzafame
+
+Last week:
+
+- env var config discussion and PR
+- get back into metrics/tracing PR
+- extending benchmarks on go-libp2p-peerstore
+- started work on ipfs-cluster iptb docker plugin
+- reviewing tf code for eks infra
+
+This week:
+
+- start documenting metrics/tracing configuration
+- bubble up latest release of opencensus before we merge metrics/tracing PR
+- clean up helm charts
+- document helm/kubernetes deployment 
+
+
+#### @mhz
+
+Last week: 
+
+- meetings with all the people
+  - tried to support David and Molly's IPFS roadmapping
+  - Matt Z on decentralized data stewardship next steps
+
+This week:
+
+- finish roadmapping stuff
+- plan protoyping process/team?? <- came out of chat with Matt Z
+  - Pick up the ideas behind Cube?
+
+- Notes for later
+  - Could relate to Federation and colaborative pinsets
+  - Come up with a way to define Cluster as a product
+  - "We support tracking XYZ"
+  - Document that defines the difference between IPFS, Cluster, and Filecoin
+  - Trying to look long-term to IPFS Conf -- would like these use cases fleshed out -- sessions on IPFS Cluster for collaborative pinning; IPFS Cluster for dataset backup; IPFS Cluster for collaboration between organization (or whatever we call that)
+  - Infra is going to have an automated deploy Cluster to AWS
+  - At the same time, we have people coming to us asking to pin things. We could say 'sure, here's a TB'. We should have a policy about who we should pin/support. What things should we look into? Perhaps, is the project open source? Are they going to store data in a way we support? How much is it going to cost? Are they going to work with us in a way that we'll learn from? We should have this in a doc so we can refer to it when people ask.
+  - Hector thinks we'll have all the major pieces together, but want to have the small pieces of the cases together, too. Want this by IPFS Conf -- Jun/Jul.
+  - MHz question: how much will we need a pinning service to be a product? Hector: Cluster fulfills many of the needs of a pinning service. But it should not do both. MHz: not sure if we'll need that, too.
+
+
+### Questions, announcements and comments
+
+* How do you...?
+* What is...?
+
+### Other business
+
+
+---
+## Standup meeting - 2018-10-23
+
+### Updates since last time
+
+##### @hsanjuan
+
+Last week:
+
+- Headers for API
+- libp2p upgrade 
+- Tests with race (more failures and report)
+- Fetch gx from ipfs
```

Comment:
What does this mean?

Reply:
https://github.com/ipfs/ipfs-cluster/pull/581